packages/validator/src/util/externalSignerClient.ts

```diff
@@ -110,7 +110,12 @@ export async function externalSignerGetKeys(externalSignerUrl: string): Promise<
     headers: {"Content-Type": "application/json"},
   });
 
-  return handlerExternalSignerResponse<string[]>(res);
+  if (!res.ok) {
+    throw Error(`${await res.text()??res.statusText}`);
```

Comment:
You are checking if body is undefined, not if it's empty in terms of an empty string
```suggestion
    throw Error(`${await res.text() || res.statusText}`);
```

Reply:
That's embarrassing 😳 Sorry.